site/en/docs/extensions/mv3/declare_permissions/index.md

```diff
@@ -329,6 +329,9 @@ The following table lists the currently available permissions:
         many circumstances your extension will not need to declare the <code>"tabs"</code> permission to make use of
         these APIs.</td>
     </tr>
+    <tr id="offscreen">
+      <td><code>"offscreen"</code></td>
+      <td>Gives your extension access to the <a href="/docs/extensions/reference/offscreen/">chrome.offscreen</a> API, allowing developers to create off-screen documents to utilize DOM APIs without obtrusively opening new windows or tabs that interrupt the user experience. Off-screen documents do not have the same APIs as the rest of the extension, but have access to the [chrome.runtime](/docs/extensions/reference/runtime/) messaging API to communicate with the service worker. 
```

Comment:
```suggestion
      <td>Gives access to the <a href="/docs/extensions/reference/offscreen/">chrome.offscreen</a> API, allowing developers to create off-screen documents to use DOM APIs without obtrusively opening new windows or tabs that interrupt the user experience. Off-screen documents do not have the same APIs as the rest of the extension, but have access to the [chrome.runtime](/docs/extensions/reference/runtime/) messaging API to communicate with the service worker. 
```

Reply:
These changes are two writing best practices.
* 'your extension'. This follows the best practice to omit needless words. The context already gives the reader the information that you're talking about an extension.
* 'utilize' to 'use'. 'Utilize' is what's called a needless variant. Avoid them.